cfgov/v1/tests/models/test_handlers.py

```diff
@@ -0,0 +1,72 @@
+import mock
+from collections import OrderedDict
+from unittest import TestCase
+from ...models.handlers import Handler, JSHandler
+
+
+class TestHandler(TestCase):
+    def setUp(self):
+        self.page = mock.Mock()
+        self.request = mock.Mock()
+        self.handler = Handler(self.page, self.request)
+
+    def test_process_raises_NotImplementedError(self):
+        with self.assertRaises(NotImplementedError) as nie:
+            self.handler.process({})
+
+    def test_get_streamfield_blocks_returns_list(self):
+        assert type(self.handler.get_streamfield_blocks()) is list
```

Comment:
I think @vickumar1981 might mean [this](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertIsInstance), e.g.

```sh
self.assertIsInstance(self.handler.get_streamfield_blocks(), list)
```